lib/compiler/src/compile.erl

```diff
@@ -719,7 +719,9 @@ core_passes() ->
 
 kernel_passes() ->
     %% Optimizations that must be done after all other optimizations.
-    [{pass,sys_core_bsm},
+    [{pass,sys_core_alias},
```

Comment:
Regarding the placement of the pass: Any reason that sys_core_alias in kernel_passes() and not in core_passes()?